bool dropped
bool pick
string object_name
string issue
